src/services/app-versions-service.ts

```diff
@@ -33,7 +33,11 @@ export const listAppVersionsByAppId = async (appId: AppId): Promise<Array<AppVer
   }
 };
 
-export const defaultVersionByAppId = async (appId: AppId, useLiveVersion = false): Promise<AppVersion | undefined> => {
+export const defaultVersionByAppId = async (
+  appId: AppId,
+  customLogMessage?: string,
+  useLiveVersion = false,
```

Comment:
This is a little problematic. multiple optional/default parameters make it harder to call this function. instead, use an `options` parameter to contain these (and all future optional params aimed at running configuration)